Solr/5/extras/solrconfig.xml

```diff
@@ -810,7 +810,7 @@
         <lst name="defaults">
             <str name="echoParams">explicit</str>
             <int name="rows">10</int>
-            <str name="df">_text</str>
+            <str name="df">text</str>
```

Comment:
The rename is good, but I'm mildly afraid it might clash at some point with the `Text` DBObject from SilverStripe, causing confusion in naming etc.. Do you have any thoughts about that?